app/controllers/manageBusinesses/cease/IncomeSourceCeasedBackErrorController.scala

```diff
@@ -42,9 +41,9 @@ class IncomeSourceCeasedBackErrorController @Inject()(val authorisedFunctions: A
 
 
   def handleRequest(isAgent: Boolean, incomeSourceType: IncomeSourceType)
-                   (implicit user: MtdItUser[_]): Future[Result] = withSessionData(JourneyType(Cease, incomeSourceType), journeyState = CannotGoBackPage) { _ =>
+                   (implicit user: MtdItUser[_]): Future[Result] = withIncomeSourcesFS(
```

Comment:
This attempts to replicate the behaviour in the original income sources journey. Since there is now a combined landing page, session data was automatically cleared on this page and caused this bug.